src/index.html

```diff
@@ -5,8 +5,339 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header container">
+      <img
+      class="header__logo"
+      src="./images/logo.png"
+      alt="logo"
+      >
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__list_item">
+            <a class="nav__list_link is-active nav__list_link--active" href="#apple">apple</a>
```

Comment:
Перенось на новий рядок текст, який в середині тегів. Так їх буде простіше читати
```suggestion
            <a class="nav__list_link is-active nav__list_link--active" href="#apple">
              Apple
            </a>
```